### PR TITLE
Don't download IPA in integration tests

### PR DESCRIPTION
**Pull request by macaptain**
Instead, IPA is bundled with the image. We don't need to run the ipa-downloader again, so set the IPA_DOWNLOAD_ENABLED environment variable to "false" to skip the download.

TODO: Run integration tests after https://github.com/metal3-io/baremetal-operator/pull/957 is merged, and check that ipa-downloader isn't run in the build output.

### CODE REVIEW

**Review by Rozzii — APPROVED** [review]
It looks good to me, and as we have discussed it will be  merged after https://github.com/metal3-io/baremetal-operator/pull/957 has been already merged.

**Comment by fmuyassarov** [comment]
This is going to fail until IPA is not injected into the image. Shall we hold this @macaptain ?
/approve

**Comment by fmuyassarov** [comment]
Feel free to unhold if the image already includes the IPA
/hold

**Comment by macaptain** [comment]
> This is going to fail until IPA is not injected into the image. Shall we hold this @macaptain ?
> /approve

IPA is already in the image, from the `make install_requirements` line in the image building scripts. (https://github.com/Nordix/airship-dev-tools/blob/16bb53af241b67176c4fb446ad01be2c614d8e89/ci/scripts/image_scripts/provision_metal3_image.sh#L56)

At the moment, for each integration test, I think we download IPA _3_ times. Once in image build, once in 01 script, and again in run_local_ironic script via BMO. Once @Rozzii's PR is merged I want to run an integration test on this PR and check:

1) Integration test is green
2) ipa-downloader is not run in the integration test

Happy to hold until then.

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**
